scripts/utils/migrate/fetch-redirects.js

```diff
@@ -0,0 +1,35 @@
+const fetch = require('node-fetch');
+const logger = require('../logger');
+const { paginate } = require('./docs-api');
+
+require('dotenv').config();
+
+const fetchRedirects = async (nodeId) => {
+  const url = `https://docs-dev.newrelic.com/api/migration/redirects?id=${encodeURIComponent(
+    `node/${nodeId}`
+  )}`;
+
+  try {
+    const res = await fetch(url, {
+      headers: {
+        'Content-Type': 'application/json',
+        'X-Phpshield-Key-Disable': process.env.ACQUIA_DEV_PHP_SHIELD_KEY,
+      },
+    });
+    const json = await res.json();
+    const { redirects } = json;
+    return redirects;
```

Comment:
Could be simplified to:
```suggestion
    const json = await res.json();
    return json.redirects;
```

Reply:
Removed this method from the module after refactor